src/components/layouts/Footer.jsx

```diff
@@ -1,6 +1,6 @@
 const Footer = () => {
   return (
-    <footer className="flex w-screen  flex-row items-center text-center justify-center bg-lightGreen w-full h-24  text-black">
+    <footer className="flex w-screen  flex-row items-center text-center justify-center bg-lightGreen h-24  text-black">
```

Comment:
Please delete from classnames w-screen; add w-full after bg-lightGreen and change the height from h-24 to h-20

Reply:
I did that and now I have narrow footer line is this ok?